packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## NEXT
+## 2.6.0
 
+* Adds option to configure HTTP headers via `VideoPlayerController` to fix access to M3U8 files on Android.
+* Adds option to configure the User-Agent in the HTTP headers.
```

Comment:
This line shouldn't be here; there's no change in `video_player` that does this. It should only be in the `video_player_android` README.